src/libraries/System.Net.Sockets/tests/FunctionalTests/TcpListenerTest.cs

```diff
@@ -44,6 +44,30 @@ public void Active_TrueWhileRunning(int ctor)
             Assert.False(listener.Active);
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsWindows))]
+        public void AllowNatTraversal_NotStarted_SetSuccessfully()
+        {
+            var listener = new TcpListener(IPAddress.Loopback, 0);
+            listener.AllowNatTraversal(true);
+        }
```

Comment:
Should we start it after calling AllowNatTraversal to confirm that still works?

Reply:
Yeah, you are right. I will add Start.